client/src/app/shared/lab-card/lab-card.component.html

```diff
@@ -1,21 +1,29 @@
 <mat-card *ngIf="lab">
-  <a class="ml-lab-card-header" title="View lab {{ lab.name}}" [routerLink]="['/editor', lab.id]">
+  <div class="ml-lab-card-header" title="View lab {{ lab.name}}" >
+    <a class="ml-lab-card-link-area" [routerLink]="['/editor', lab.id]"></a>
```

Comment:
This link however, needs the title :)

Reply:
I noticed that this link takes the whole labcard into account, not just the header. This is a problem because all executions attached to this card are no longer linkable. 

I'd rather vote for removing the link area thing entirely and just go with the multiple links, at the cost of not having the whole title area clickable.

Reply:
@PascalPrecht so, setting the `position: relative` to `ml-lab-card-header` solved it, because without this the `ml-lab-card-link-area` will take up the whole space of the next `relative` positioned element which is the `<mat-card>` that contains the executions as well.